app/grandchallenge/reader_studies/forms.py

```diff
@@ -492,6 +494,8 @@ def clean_ground_truth(self):
 
 class DisplaySetCreateForm(Form):
     _possible_widgets = {
+        SelectUploadWidget,
+        autocomplete.ModelSelect2,
```

Comment:
How does this form produce those widgets?

Reply:
Good point, it doesn't.